mobile/test/common/BUILD

```diff
@@ -10,6 +10,7 @@ envoy_cc_test(
     repository = "@envoy",
     deps = [
         "//library/common:engine_common_lib",
+         "@envoy//source/extensions/listener_managers/listener_manager:listener_manager_lib",
```

Comment:
Shouldn't this dep go in `//library/common:engine_common_lib` since it should always be compiled in, not just for tests?